api/flaskr/__init__.py

```diff
@@ -111,16 +119,52 @@ def login():
         if login_user:
             if bcrypt.check_password_hash(login_user['password'], password):
                 session['username'] = username
-                session['access_token'] = create_access_token(identity=username)
-                session['refresh_token'] = create_access_token(identity=username)
-                
-                return redirect(url_for('index'))
+                expires = datetime.timedelta(hours=1)
```

Comment:
Let's increase expiration time, it gets annoying to login again and again. Maybe we can have an option called 'keep me signed in' and send that value.

Reply:
Changed it on this [commit](https://github.com/Dogacel/Kalas-Iris/pull/35/commits/43e55a3eb731d414dab39037acf3b9ed6f690131)